CMakeLists.txt

```diff
@@ -27,3 +27,48 @@ add_subdirectory(src)
 
 # Examples
 add_subdirectory(example)
+
+libp2p_install(libp2p)
+
+libp2p_install(p2p_varint_prefix_reader)
```

Comment:
Maybe better to put libp2p_install under the first declaration of the each library?

Reply:
Will replace all `add_library` to `add_library_libp2p_install`